test/__init__.py

```diff
@@ -1,4 +1,3 @@
 import sys
 
-
```

Comment:
czemu zmniejszyłaś odstęp między importem a kodem? czy to jest wina pre-commita?